Remove new messages logic and models and models plug

AndreyTepin opened the pull request.

AndreyTepin committed:
intermediate commit and update Connection Docker

AndreyTepin committed:
The commit implemented the deletion of messages with a mention, in 10 seconds after they were sent, from all users, except for the Bot.

where-kyfat commented:
Functional don't work

AndreyTepin committed:
Fix logic remove

where-kyfat commented on requirements.txt:
```diff
@@ -1 +1,3 @@
 discord==1.7.3
+peewee==3.14.9
+psycopg2==2.9.3
```
Don't understand where y use it
```suggestion
```

where-kyfat commented on models.py:
```diff
@@ -0,0 +1,28 @@
+import peewee as pw
+
+
+db = pw.PostgresqlDatabase(
+    'postgres',
+    user='postgres',
+    password='postgres',
+    host='db',
+    port='5432'
+)
+
+
+class BaseModel(pw.Model):
+    class Meta:
+        database = db
```
I think this is a base plug for future models. Please reflect this in pull request message

where-kyfat commented on main.py:
```diff
@@ -11,4 +11,12 @@ async def hello(ctx):
+async def on_message(message):
+    await bot.process_commands(message)
+
+    if message.mentions != []:
```
Python checking for emptiness object by this way
```suggestion
    if message.mentions:
```

where-kyfat commented on models.py:
```diff
@@ -0,0 +1,28 @@
+class BaseModel(pw.Model):
+    class Meta:
+        database = db
+
+
+class Messages(BaseModel):
+    id = pw.PrimaryKeyField(null=False)
+    message = pw.CharField(max_length=100)
+    user = pw.CharField(max_length=100)
+    #dis_id = pw.IntegerField(default=123)
+    dis_id_str = pw.CharField(max_length=100)
+
+    created_at = pw.DateTimeField(default=pw.datetime.datetime.now())
+    updated_at = pw.DateTimeField(default=pw.datetime.datetime.now())
```
I think we don't need it now
```suggestion
```

where-kyfat commented on operations.py:
```diff
@@ -0,0 +1,15 @@
+import peewee
+from models import *
+
+
+def add_message(dis_id, message, author):
+    row = Messages(
+        dis_id_str="{}".format(dis_id),
+        message=message.strip()+"qwe",
+        user=author
+    )
+    row.save()
+
+
+def find_message(dis_id):
+    return Messages.get("{}".format(dis_id)==Messages.dis_id_str)
```
We don't use it. Please, remove
```suggestion
```

where-kyfat reviewed (changes requested):
Some rebases needed

AndreyTepin committed:
Update requirements.txt

Co-authored-by: Олег Шабаршин <oleg.shabarshin@gmail.com>

AndreyTepin committed:
Review changes

https://github.com/where-kyfat/discordBot_hello/pull/3#pullrequestreview-900138957

AndreyTepin committed:
Merge remote-tracking branch 'origin/db_logic' into db_logic
